nemo_text_processing/text_normalization/taggers/tokenize_and_classify.py

```diff
@@ -83,14 +84,10 @@ def __init__(self, input_case: str, deterministic: bool = True):
             | pynutil.add_weight(money_graph, 1.1)
             | pynutil.add_weight(telephone_graph, 1.1)
             | pynutil.add_weight(electonic_graph, 1.1)
+            | pynutil.add_weight(serial_graph, 1.2)
```

Comment:
why having higher weight?

Reply:
to choose the serial class only if cardinal and the rest of the semiotic classes fail to tag the text, otherwise the serial is chosen and the default tests are failing